spec/views/shared/header_navbar.html.erb_spec.rb

```diff
@@ -1,17 +1,5 @@
 # frozen_string_literal: true
 describe 'catalog/index', type: :system do
-  let(:recently_added) { file_fixture("recent.json") }
```

Comment:
We don't need this stub now that the code it not issuing a call to `/catalog.json` for the recent feeds.